File: Python Code/Daily_Flash_Code_0/Week9/20Mar/52-DailyFlash_Solutions/20-Mar/Java/prog5.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		int n, sum = 0, num;
		Scanner sc = new Scanner(System.in);
		n = sc.nextInt();
		int [] arr = new int[n];
		for(int i = 0; i < n; i++) {
			arr[i] = sc.nextInt();
			sum += arr[i];
		}
		float avg = (float)sum / (float)n;
		sum = 0;
		for(int i = 0; i < n; i++) {
			sum += (avg - arr[i]) * (avg - arr[i]);
		}
		System.out.println((float)sum / n);
	}
}

File: Python Code/Daily_Flash_Code_0/Week9/20Mar/52-DailyFlash_Solutions/20-Mar/C++/prog5.cpp
#include<iostream>

int main() {
	int n, sum = 0, num;
	std::cin >> n;
	int arr[n];
	for(int i = 0; i < n; i++) {
		std::cin >> arr[i];
		sum += arr[i];
	}
	float avg = (float)sum / (float)n;
	sum = 0;
	for(int i = 0; i < n; i++) {
		sum += (avg - arr[i]) * (avg - arr[i]);
	}
	std::cout << (float)sum / n;
}
